front/src/applications/operationalStudies/components/MacroEditor/osrdToNge.ts

```diff
@@ -177,6 +183,31 @@ const castNodeToNge = (
   ),
 });
 
+export const getTrainrunCategories = (t: TFunction): TrainrunCategory[] => {
+  const mappedCategories: TrainrunCategory[] = Object.entries(OSRD_TRAINRUN_CATEGORY_MAPPING).map(
+    ([key, category]) => ({
+      ...category,
+      name: t(`macroEditor.trainCategory.${key}.name`),
+      shortName: t(`macroEditor.trainCategory.${key}.shortName`),
+    })
+  );
+
+  const noCategory: TrainrunCategory = {
+    id: 10,
+    order: 11,
+    name: t('macroEditor.trainCategory.NO_CATEGORY.name'),
+    shortName: t('macroEditor.trainCategory.NO_CATEGORY.shortName'),
```

Comment:
```suggestion
    name: t('main.macroEditor.trainCategory.NO_CATEGORY.name'),
    shortName: t('main.macroEditor.trainCategory.NO_CATEGORY.shortName'),
```

I see that some translation keys have moved recently, and now use `main`

in `getNgeTrainrunFrequencies()`:
```ts
          name: t('main.macroEditor.intervalXmin', { minutes: intervalInMinutes }),
```

It might be useful to check if you are aligned with this.

Reply:
Indeed, what I did also affected the frequencies. It's fixed now.